src/styles/app/10-settings-controls.css

```
@@ -35,20 +35,55 @@
 	background: color-mix(in srgb, var(--color-blue-500) 10%, transparent);
 }
 
+.earlyAccessBadge {
+	--early-access-color: var(--interactive-accent);
+	padding: 3px 8px;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)